### PR TITLE
LPD-28831 | Master

### PR DESCRIPTION
**Pull request by interaminense**
## How to change analytics client js CDN version

With the introduction of the [analytics-internal.liferay.com](http://analytics-internal.liferay.com/), and the new canary release process we want to completely isolate this environment build and artifacts from the other production environments.

This includes the analytics.js code pointing to the latest published CDN version.

With this new feature, we aim to have the JS SDK URL configurable so all the internal DXPs connected to Analytics Cloud point to a release candidate version.

We will change the analytics client js CDN version via `portal.properties`.
There are two ways to change the property value:

### Via portal-ext.properties
Add `analytics.cloud.client.js.version=INTERNAL` to portal-ext.properties.

### Via UI on DXP
1. Go to Control Panel > Server Administrator > Script
2. Execute the following code:
```
import com.liferay.portal.util.PropsUtil;

PropsUtil.set("analytics.cloud.client.js.version", "INTERNAL");
```

| Options | Points to |
|--------|--------|
| PROD | https://analytics-js-cdn.liferay.com |
| INTERNAL | https://analytics-js-internal-cdn.liferay.com |
| DEV | https://analytics-js-dev-cdn.liferay.com |

Documentation https://liferay.atlassian.net/wiki/spaces/ENGAC/pages/2949775389/How+to+change+analytics+client+js+CDN+version

### CODE REVIEW

**Comment by interaminense** [comment]
cc: @marcellustavares

**Comment by interaminense** [comment]
ci:test:sf

**Comment by interaminense** [comment]
ci:test:sf

**Comment by interaminense** [comment]
ci:test:relevant

File: portal-impl/src/portal.properties [hunk 1]
```diff
@@ -10276,6 +10276,17 @@
...
+##
+## Analytics Cloud
+##
+
+    #
+    # Analytics Cloud SDK Version
+    #
+    # Env: LIFERAY_ANALYTICSCLOUD_PERIOD_SDK_PERIOD_VERSION
+    #
+    analyticscloud.sdk.version=PROD
```
Comment (marcellustavares) [inline portal-impl/src/portal.properties:10288]: Can we use `analytics.cloud.client.js.version` ? So it's closer to its [module name](https://github.com/liferay-ac/liferay-portal/tree/master/modules/apps/analytics/analytics-client-js)?

**Review by marcellustavares — CHANGES_REQUESTED** [review]

File: portal-impl/src/portal.properties (hunk 1 above)
Comment (interaminense) [reply portal-impl/src/portal.properties:10288]: Sure thing, thanks @marcellustavares

**Comment by ViniLopes20** [comment]
ci:test:analytics-cloud-acceptance

**Comment by interaminense** [comment]
ci:test:sf

**Comment by interaminense** [comment]
ci:test:relevant

**Comment by interaminense** [comment]
ci:test:sf

**Comment by interaminense** [comment]
ci:test:relevant

**Comment by interaminense** [comment]
ci:test:relevant

**Comment by interaminense** [comment]
ci:test:relevant

**Comment by interaminense** [comment]
ci:test:relevant

**Comment by interaminense** [comment]
ci:test:sf

**Comment by interaminense** [comment]
ci:forward

**Pull request closed without merging**